library/src/main/kotlin/com/chuckerteam/chucker/api/Chucker.kt

```diff
@@ -5,13 +5,19 @@ import android.content.Intent
 import android.content.pm.ShortcutInfo
 import android.content.pm.ShortcutManager
 import android.graphics.drawable.Icon
+import android.net.Uri
 import android.os.Build
 import android.util.Log
+import android.widget.Toast
 import androidx.core.content.getSystemService
 import com.chuckerteam.chucker.R
+import com.chuckerteam.chucker.internal.data.repository.RepositoryProvider
```

Comment:
I don't think we need these imports as well. Feels like some leftovers from experimenting, etc.